### PR TITLE
geocoder: Implement OlaMaps Reverse geocoder

### PR DESCRIPTION
**Pull request by mujeerhashmi**
We have integrated Ola Maps geocoder. Kindly review.

### CODE REVIEW

File: src/main/java/org/traccar/geocoder/OlaMapsGeocoder.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        }
+        return url;
+    }
+
+    public OlaMapsGeocoder(Client client, String key, int cacheSize, AddressFormat addressFormat) {
+        super(client, formatUrl(key), cacheSize, addressFormat);
+    }
+
+    @Override
+    public Address parseAddress(JsonObject json) {
```
Comment (tananaev) [inline src/main/java/org/traccar/geocoder/OlaMapsGeocoder.java:38]: It seems like format matched Google geocoder, so it doesn't make sense to duplicate it.

Comment (mujeerhashmi) [reply src/main/java/org/traccar/geocoder/OlaMapsGeocoder.java:38]: Yes it's almost the same. How can we use this geocoder which has different url.

Comment (tananaev) [reply src/main/java/org/traccar/geocoder/OlaMapsGeocoder.java:38]: You can probably modify Google geocoder to allow custom URL. But duplicating code is never a good solution.

Comment (mujeerhashmi) [reply src/main/java/org/traccar/geocoder/OlaMapsGeocoder.java:38]: Ok will modify the PR.

Comment (mujeerhashmi) [reply src/main/java/org/traccar/geocoder/OlaMapsGeocoder.java]: > You can probably modify Google geocoder to allow custom URL. But duplicating code is never a good solution.

I have done these changes. Kindly review.

**Comment by mujeerhashmi** [comment]
@tananaev done

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**